tests/antares/services/local_services/test_study.py

```diff
@@ -1498,7 +1498,7 @@ def test_creating_duplicate_area_name_errors(self, local_study_w_areas):
         # Then
         with pytest.raises(
             AreaCreationError,
-            match=f"Could not create the area {area_to_create}: There is already an area '{area_to_create}' in the study '{local_study_w_areas.name}'",
+            match=f"Could not create the area {area_to_create}: There is already an area '{area_to_create}' in the study '{local_study_w_areas.name}",
```

Comment:
This shouldn't change i think as you have only one ` at the beginning but not in the end